Ginger/GingerCore/Drivers/JavaDriverLib/GingerJavaAgent/agent/com/amdocs/ginger/PayLoad.java

```diff
@@ -620,12 +635,13 @@ public void DumpToConsole()
         GingerAgent.WriteLog(s);
     }
     
-    public static PayLoad Error(String ErrorMessage) 
+    public static PayLoad Error(String errorMessage,int errorCode) 
```

Comment:
Argument sequence should be same in all payload classes i.e. C# java and JS side

Error code and then error message